### DISCUSSION TITLE
Iconify support

### DISCUSSION THREAD

**Discussion by thomasnavarro** [flyntwp/flynt#505] [Ideas]
Hi there,

I love using Iconify to manage my icons, I've created a PR #504 to implement it in Flynt.

If you have any feedback !

**Comment by timohubois** [comment]
Hey @thomasnavarro, thanks for the idea and the PR. We have discussed this topic with the team. You can find details and a feedback here: https://github.com/flyntwp/flynt/pull/504#issuecomment-1691584566
